src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let attempts = 0;
+
+  return function(a) {
+    attempts++;
+
+    return function(b) {
+      return attempts > 3 && attempts % 2 === 0 ? 'Bzzz... Error!'
+        : a + b;
```

Comment:
Ternary looks amazing, I love it, too, but let's use simple if-statement – it makes your code more readable